src/Components/CreateWorkflow/CreateWorkflow.tsx

```diff
@@ -1,4 +1,6 @@
 import React from "react";
```

Comment:
Nice moving these out to Components dir to be shared. I don't think there is an easy way around doing the different checks without adding some abstraction that would ultimately be more confusing than the two types